src/components/auth/guard/PaymentAccessGuard.tsx

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from '@tanstack/react-query';
+import memberApi from '@/apis/member/memberApi';
+import { useNavigate } from 'react-router-dom';
+import RoutePath from '@/routes/routePath';
+
+const PaymentAccessGuard = () => {
+  const navigate = useNavigate();
+  const { data } = useQuery({
+    queryKey: ['member'],
+    queryFn: memberApi.GET_DASHBOARD
+  });
+
+  if (!data) {
+    return <div>로딩중..</div>;
+  }
+  if (
+    data.member.role !== 'ASSOCIATE' ||
+    !data.currentRecruitmentRound.period.open
+  ) {
+    navigate(RoutePath.Dashboard);
+  }
+};
+
+export default PaymentAccessGuard;
```

Comment:
p3;
return 값으로 `<Outlet/>` 이 있어야 조건을 만족했을 때에는 페이먼츠 화면이 나올 것 같아요!
```suggestion
  }
  
  return <Outlet/>
};

export default PaymentAccessGuard;
```